package.json

```
@@ -19,7 +19,7 @@
     "README.md"
   ],
   "engines": {
-    "node": "^8"
+    "node": ">8"
```

```suggestion
    "node": ">= 8"
```

`> 8` doesn't include 8.

---

😭